lib/improver/tests/wxcode/wxcode/test_wxcode_utilities.py

```diff
@@ -93,6 +96,21 @@ def setUp(self):
                          2, 0, 1, 29, 30, 1, 5, 6, 6]).reshape(2, 1, 3, 3)
         self.cube = set_up_cube(data, 'air_temperature', 'K',
                                 realizations=np.array([0, 1]))
+        self.wxcode = np.array(sorted(WX_DICT.keys()))
+        self.wxmeaning = ''
+        for i, val in enumerate(self.wxcode):
+            if i == len(self.wxcode)-1:
+                self.wxmeaning += WX_DICT[val]
+            else:
+                self.wxmeaning += WX_DICT[val] + ' '
+        self.data_directory = mkdtemp()
+        self.nc_file = self.data_directory + '/wxcode.nc'
+        Call(['touch', self.nc_file])
```

Comment:
Do you need to touch the file, as won't it just be created anyway when you save the file? Or is this for dealing with when something doesn't work as expected?

Reply:
Added the touch because there are two tests that do not create the file and I was getting "rm No such file or directory" messages when running the unit tests

Reply:
OK. I thought there might be a reason like that.